apps/designer/app/routes/rest/pages.$projectId.ts

```diff
@@ -3,7 +3,11 @@ import { db } from "@webstudio-is/project/index.server";
 import { type Pages, utils } from "@webstudio-is/project";
 import { zfd } from "zod-form-data";
 import { z } from "zod";
-import { CreatePageData } from "~/designer/features/sidebar-left/panels/pages/settings";
+import {
+  CreatePageData,
+  DeletePageData,
+  EditPageData,
+} from "~/designer/features/sidebar-left/panels/pages/settings";
```

Comment:
1. we never import things from features except of inside the component that puts features together, like Designer
2. when there is an index.ts, we never reach into that folder to get random files, index.ts acts as a public interface, everything it didn't export is private. There is a linter for this we haven't installed yet, can you please create a ticket to add it?

You probably want to move those types to some shared location and make sure that we don't import the js from that module if we only need types. I was wondering sometimes if our compiler is smart enough to do that or do we need to be careful and put `type` keyword after import